mapie/tests/test_utils.py

```diff
@@ -410,3 +410,22 @@ def test_change_values_zero_one() -> None:
     """Test that binary output are changed to zero one outputs."""
     array_ = check_binary_zero_one(np.array([0, 4, 4]))
     assert (np.unique(array_) == np.array([0, 1])).all()
+
+
+# Valid Gamma Test
+def test_valid_gamma():
+    valid_gamma = 0.5
+    assert check_gamma(valid_gamma) is None
+
+
+# Invalid Gamma Tests
+def test_invalid_negative_gamma():
+    negative_gamma = -0.1
+    with pytest.raises(ValueError, match="Gamma must be between 0 and 1."):
+        check_gamma(negative_gamma)
+
+
+def test_invalid_large_gamma():
+    large_gamma = 1.5
+    with pytest.raises(ValueError, match="Gamma must be between 0 and 1."):
+        check_gamma(large_gamma)
```

Comment:
STYLE: merge these tests and add the comments in the description of the function
```suggestion
@pytest.mark.parametrize("gamma", [1.5, -0.1])
def test_invalid_large_gamma(gamma: float) -> None:
    """Test a non-valid gamma parameter."""
    with pytest.raises(ValueError, match="Gamma must be between 0 and 1."):
        check_gamma(gamma)
```